cmd/parse.go

```diff
@@ -77,5 +97,7 @@ func parse(args []string, stdout io.Writer, stderr io.Writer) int {
 
 func init() {
 	parseCommand.Flags().VarP(parseParams.format, "format", "f", "set output format")
+	parseCommand.Flags().StringVarP(&parseParams.jsonOptions, "json-options", "o", "", "select optional elements, current options: locations, comments. E.g. -o locations,-comments will include locations and exclude comments.")
+
```

Comment:
This looks nice!

Reply:
Agreed, the doc string is very helpful there: I was able to immediately understand how to use the new options format. :+1:

Reply:
My only hesitation is that `-o` _usually_ means some sort of output in all the CLI tools I remember 🤔

Reply:
Yeah, that's a fair point. I think the flag can be renamed to `--include`

Reply:
This has now been updated to `--json-include` to make it clear that it's not an output format, and to make it clear that the option only applies to the json output format.